src/Queue/RabbitMQQueue.php

```diff
@@ -525,6 +525,11 @@ protected function createMessage($payload, int $attempts = 0): array
         }
 
         if (isset($currentPayload['data']['command'])) {
+            // If the command data is encrypted, decrypt it first before attempting to unserialize
```

Comment:
`is_subclass_of` should work here

Reply:
@khepin sorry, what do you mean? I'm not following

Reply:
@khepin Sorry, I'm not following, what are you referring to here?

Reply:
Sorry, put the comment on the wrong line. You should be able to replace `in_array(ShouldBeEncrypted::class, class_implements($commandName))` with `is_subclass_of($commandName, ShouldBeEncrypted::class)` on the line below this one.

Reply:
Huh, I didn't realize that worked on interfaces. Learn something new every day!